docs/k8s_deploy.md

```diff
@@ -0,0 +1,162 @@
+<!--
+Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Deploying Model Analyzer on a Kubernetes cluster
+
+Model Analyzer provides support deployment on a Kubernetes enabled
+cluster using helm charts. You can find information about helm charts [here](https://helm.sh/).
+
+## Requirements
+
+* A Kubernetes enabled cluster
+* Helm
```

Comment:
would be nice to have links to references for both of these requirements.

Reply:
Should I put links on this line? Because the link for helm is in the section above and the link for Kubernetes is the first point in the section below.